app/views/admin/reports/_deal.html.erb

```diff
@@ -1,5 +1,5 @@
 <td><%= "Deal #{deal.id}" %></td>
 <td><%= deal.title %></td>
-<td><%= price_in_currency(deal.deal_price_with_tax * deal.orders_count) %></td>
-<td><%= price_in_currency(deal.deal_price_with_tax) %>
-<td><%= price_in_currency(deal.deal_price_with_tax * (deal.quantity + deal.orders_count)) %>
+<td><%= price_in_dollar(deal.deal_price_with_tax * deal.orders_count) %></td>
```

Comment:
let's show reports in user's currency preference instead

Reply:
done